site/content/docs/install_collector.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "SNAS Collector Docker Install"
+date: 2017-05-07T15:04:10.000Z
+---
+
+SNAS Collector Docker Install Steps
+==============================================
+
+<!--more-->
+
+Collector container is the container for collecting BMP messages from BMP senders, e.g. routers. This container can be distributed.
+
+### Container Includes
+* **Openbmpd** - Latest collector (listening port is TCP 5000)
+
+### Recommended Current Linux Distributions
+
+  1. Ubuntu 14.04/Trusty
+  1. CentOS 7/RHEL 7
+
+### 1) Install docker
+Docker host should be **Linux x86_64**.   Follow the [Docker Instructions](https://docs.docker.com/installation/) to install docker.  
+
+- - -
+
+### 2) Download the docker image
+
+    docker pull openbmp/collector
+
+- - -
+
+### 3) [OPTIONAL] Add persistent configs
+
+#### On host create persistent config location
+
+    mkdir -p /var/openbmp/config
+    chmod 777 /var/openbmp/config
+
+#### config/hosts
```

Comment:
Added this "You can also add other hosts into a container’s /etc/hosts file by using one or more --add-host flags. " under "config/hosts". If you have an example, we can add that too.